tests/benchmark/bm_files.sh

```diff
@@ -32,4 +32,4 @@ elif [ "$BM_TYPE" = "bm-updated-fp32-single" ]; then
     file_name="updated"
 fi
 
-wget --no-check-certificate -q -i tests/benchmark/data/hnsw_indices/hnsw_indices_$file_name.txt -P tests/benchmark/data
```

Comment:
Intentionally removing the `-q`?

Reply:
Yes, it doesn't affect the time and might be useful if something is wrong